components/nds-panel/src/PanelItem.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import classnames from "classnames";
+import PropTypes from "prop-types";
+
+export const PanelItem = ({ children, className, icon } = {}) => {
+	const classes = classnames("nds-panel__item", className);
```

Comment:
What's this component? It uses a `nds-panel__item` class which doesn't seem to be in a separate _paneitem.scss_ file and it's not in the main panel.scss